### DISCUSSION TITLE
Google confirms they will disable uBlock Origin in Chrome in 2024 News

### DISCUSSION THREAD

**Discussion by mms-gianni** [kubero-dev/ladder#43] [Ideas]
![image](https://github.com/everywall/ladder/assets/2052196/1edb5605-7825-4300-918b-6391bb630c99)

https://www.reddit.com/r/uBlockOrigin/comments/17wu2gz/google_confirms_they_will_disable_ublock_origin/ 

Well, ladder removes most ads, even though this was never intended. To be honest, I'm still not sure why this happens.

**Comment by deoxykev** [comment]
I think it certainly be possible to block ads using ladder. It would mean that ladder should be a full-blown socks proxy, however.

**Comment by mms-gianni** [comment]
This might be too far away from the initial service, to self-host 12ft.io

**Reply by gilgongo** [reply]
Might also be a bit too similar to https://pi-hole.net (although incorporating Ladder into **that** might be interesting...)

**Comment by deoxykev** [comment]
Adblocking might be as simple as injecting this JS library into pages...

https://github.com/ghostery/adblocker

> Cliqz' adblocker is a JavaScript library for blocking ads, trackers, and annoyances with a strong focus on [efficiency](https://whotracks.me/blog/adblockers_performance_study.html). It was designed with compatibility in mind and integrates seamlessly with the following environments:

> The library supports 99% of all filters from the Easylist and uBlock Origin projects. Check [the compatibility matrix](https://github.com/ghostery/adblocker/wiki/Compatibility-Matrix) on the wiki for more details.

**Comment by Aaron-Libra** [comment]
Good thing anyone with a brain has stopped using Chrome years ago by now (even Helium if I'm being honest...can't trust either if you consider the source -- no pun intended lol). Opera GX and, ironically, even M$ Edge are far superior. That said, I don't use Edge for much of anything really but sometimes certain sites don't cooperate with Opera (not often though -- it's likke a 0.4% problem).

Then there's the all-powerful and almighty Tor :) Obviously
